pp-2023/src/app/kmeans/calculateButton.js

```diff
@@ -8,16 +8,40 @@ export function HandleCalculateButtonClick() {
     const {numberOfClusters} = UseInputKPoints();
```

Comment:
Die Commit Message ist komplett bodenlos

Reply:
Ansonsten funktioniert das ja zumindest mit der Lokalen berechnung. Allerdings hast du dabei den "ScatterChart" vergessen. Wenn du in Zeile 67 Bitte noch die Funktion ScatterChart aufrufst dann gebe ich den bumms frei.